### PR TITLE
[stripe-v3] add definition for verifyIdentity method

### PR DESCRIPTION
**Pull request by sergey-samusenko**
add definition for verifyIdentity method

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test stripe-v3`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://stripe.com/docs/js/identity/modal

### CODE REVIEW

**Review by kaznovac — COMMENTED** [review]
the right cause of action would be to deprecate these types in favor of official types. but that requires a little bit of digging into DefinitelyTyped package publishing infrastructure...

see this discussion https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/64046

File: types/stripe-v3/index.d.ts [hunk 1]
```diff
@@ -857,6 +860,10 @@ declare namespace stripe {
         error?: Error | undefined;
     }
 
+    interface VerifyIdentityResponse {
+        error?: Error | undefined;
+    }
```
[the remark this answers is not in the document]
Comment (sergey-samusenko) [reply types/stripe-v3/index.d.ts]: thanks, updated.

**Review by kaznovac — APPROVED** [review]
Thanks @sergey-samusenko

File: types/stripe-v3/index.d.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (sergey-samusenko) [reply types/stripe-v3/index.d.ts]: my testing proved that result can have neither of them (when used with sandbox environment, success response has neither validation session id nor errors), so I'm not sure I want to be more strict with this.

**Comment by sergey-samusenko** [comment]
Ready to merge

**Pull request merged**